patientsearch/__init__.py

```diff
@@ -47,6 +50,43 @@ def create_app(testing=False):
     return app
 
 
+class LogServerHandler(logging.Handler):
+    """Specialized logging handler capable of nesting json and passing auth"""
+
+    def __init__(self, url, jwt, level):
+        super().__init__(level)
+        self.jwt = jwt
+        if url and url.endswith('/'):
+            self.url = f"{url}events"
+        else:
+            self.url = f"{url}/events"
```

Comment:
I would be less flexible with the `LOGSERVER_URL` and require it does not contain a trailing slash. It can be a pain to remember to handle/pre-process the trailing slash everywhere.

```suggestion
        self.url = f"{url}/events"
```

Reply:
hm, can we at least raise an exception if not the expected?  which is the standard (include the trailing slash, or not?), i see a sprinkling of both...

Reply:
I found [a discussion on SO](https://stackoverflow.com/questions/5948659/when-should-i-use-a-trailing-slash-in-my-url), but I don't think there's a "right" answer, just personal preference.

My personal reasoning is I think first slash in a path makes more sense at the beginning of a relative path, rather than the base URL.

Again, I can't find any compelling argument either way so feel free to leave as-is